.babelrc

```diff
@@ -12,4 +11,4 @@
     "transform-react-jsx",
     "syntax-dynamic-import"
   ]
-}
\ No newline at end of file
+}
```

Comment:
Doesn't this mean that any file under the server JS will be processed with babel? I don't think this is a good solution if that's the case. I want to avoid server side transpiling if at all possible.

Reply:
There is no transpiling, because our cli does not use babel. We only have babel when running using webpack (and jest later). If the client reach out to server code, then that would be transpiled but as we currently dont share server code and client, it doesnt happen.